.github/workflows/update-flake.yml

```
@@ -60,6 +60,10 @@ jobs:
             --flake ./flake/dev \
             --option commit-lock-file-summary "flake: update dev inputs"
 
+          nix run .#all-maintainers
+          git add stylix/generated/all-maintainers.nix
+          git commit -m "stylix: update all-maintainers list"
```

I believe this exits non-zero if there are no changes to commit.

As a slightly hacky solution, you could do:
```suggestion
          git commit -m "stylix: update all-maintainers list" ||
            echo "all-maintainers has no changes"
```

or

```suggestion
          git commit -m "stylix: update all-maintainers list" || true
```